docs-2.0/2.quick-start/4.nebula-graph-crud.md

```diff
@@ -53,6 +53,8 @@ From the **Status** column of the table in the return message, you can see that
     CREATE SPACE <graph_space_name> [(partition_num=<partition number>, replica_factor=<replica number>), vid_type=fixed_string(<string_number>))]
     ```
 
+    > **NOTE:** After the creation of a graph space, wait 20 seconds before running the `USE` statement to use the graph space.
```

Comment:
and you can also use `show spaces xxx` or `describe space xxx` to ensure the specific space is created

Reply:
besides. I'm not sure if `sleep(20)` is ok in nGQL 2.0